data/sql/updates/pending_db_world/fix-quest-9175-typo.sql

```diff
@@ -0,0 +1 @@
+UPDATE `quest_offer_reward` SET `RewardText` = 'You say that you found it on one of the Scourge at Windrunner Spire and that there\'s an inscription on it? Let me see! Here, take this coin for a job well done. ' WHERE (`ID` = 9175);
```

Comment:
This should make the two newline
```suggestion
UPDATE `quest_offer_reward` SET `RewardText` = 'You say that you found it on one of the Scourge at Windrunner Spire and that there\'s an inscription on it? Let me see! $B$B Here, take this coin for a job well done. ' WHERE (`ID` = 9175);
```